src/dymaptic.GeoBlazor.Core/Objects/NearestPointResult.cs

```diff
@@ -0,0 +1,31 @@
+﻿using dymaptic.GeoBlazor.Core.Components.Geometries;
+using dymaptic.GeoBlazor.Core.Model;
+
+
+namespace dymaptic.GeoBlazor.Core.Objects;
+
+/// <summary>
+///     Object returned from the nearestCoordinate(), nearestVertex(), and nearestVertices() methods of <see cref="GeometryEngine"/>.
+/// </summary>
+public class NearestPointResult
```

Comment:
I noticed that Geometries seem to be missing the `M` property while looking at this return.

Also, why are Geometries in `Components` not in `Model`?

I looks like the JS geometry engine has not yet caught up to the Runtime engine and doesn't have all of the linear referencing functions, but we should still track and provide the `M` values.

Reply:
Geometries is in `Components` because it can be declared as a razor component in markup.
I'm adding `M` to `Point`, I don't see that as a property on the other geometries in the JS API docs. Some of them have `hasM`, but that's calculated and shouldn't be a settable field as far as I can tell.